### PR TITLE
chore(docs): add canonical links

### PR DESCRIPTION
**Pull request by Haroenv**
**Summary**

Needed for the migration to main doc

**Result**

v2 documentation root is no longer accessible

### CODE REVIEW

**Comment by Haroenv** [comment]
It works, but don't merge this until the main docs is released obviously 👍

File: community-website/src/community-project-boilerplate-docgen/src/widgets/range-slider.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 ---
 title: Widgets - Range Slider
 layout: widget.pug
+canonical: https://www.algolia.com/doc/api-reference/widgets/range-input/angular/
```
Comment (tkrugg) [inline community-website/src/community-project-boilerplate-docgen/src/widgets/range-slider.md]:
```suggestion
canonical: https://www.algolia.com/doc/api-reference/widgets/range-slider/angular/
```

**Comment by tkrugg** [comment]
Thanks a lot!

File: community-website/package.json [hunk 2]
```diff
@@ -4,7 +4,7 @@
   "version": "1.0.0",
   "scripts": {
     "start": "NODE_ENV=development node src/community-project-boilerplate-docgen/start.js",
-    "docs:build": "NODE_ENV=production node src/community-project-boilerplate-docgen/build.js",
+    "docs:build": "NODE_ENV=production node src/community-project-boilerplate-docgen/build.js && cp _redirects docs",
```
Comment (tkrugg) [inline community-website/package.json]: I think maybe it's best that the redirection is functional only when we decide to deprecate V2, no?

Comment (Haroenv) [reply community-website/package.json]: It seems fine to me, but you're right, we can wait to redirect the root until v3 is stable?

File: community-website/src/community-project-boilerplate-docgen/src/widgets/numeric-selector.md [hunk 3]
```diff
@@ -1,6 +1,7 @@
 ---
 title: Widgets - Numeric Selector
 layout: widget.pug
+# no canonical widget because this is not replaced
```
Comment (samouss) [inline community-website/src/community-project-boilerplate-docgen/src/widgets/numeric-selector.md]: 👍

File: community-website/src/community-project-boilerplate-docgen/src/widgets/panel.md [hunk 4]
```diff
@@ -1,6 +1,7 @@
 ---
 title: Widgets - Panel
 layout: widget.pug
+canonical: https://www.algolia.com/doc/api-reference/widgets/panel/angular/
```
Comment (samouss) [inline community-website/src/community-project-boilerplate-docgen/src/widgets/panel.md]: This page does not exist. It looks like we forgot to add it to the main doc.

Comment (Haroenv) [reply community-website/src/community-project-boilerplate-docgen/src/widgets/panel.md]: What do you suggest; no canonical link or adding this in the docs first?

Comment (samouss) [reply community-website/src/community-project-boilerplate-docgen/src/widgets/panel.md]: First add the doc page, then keep the canonical link.

Comment (tkrugg) [reply community-website/src/community-project-boilerplate-docgen/src/widgets/panel.md]: Yes, I wasn't sure what to do with the panel yet. 
After our discussion, it seems clear we should almost keep it as is, so I'll do that.

Comment (tkrugg) [reply community-website/src/community-project-boilerplate-docgen/src/widgets/panel.md]: https://github.com/algolia/doc/pull/2673

**Pull request merged**